aries_vcx_core/Cargo.toml

```diff
@@ -43,7 +43,7 @@ derive_builder = "0.12.0"
 uuid = { version = "1.3.0", default-features = false, features = ["v4"] }
 tokio = { version = "1.20" }
 # TODO: Point to the official repo if / when vdr-proxy-client PR is merged: https://github.com/hyperledger/indy-vdr/pull/184
```

Comment:
The comment is outdated, the PR has been merged

Reply:
I've removed the comment altogether, created issue instead https://github.com/hyperledger/aries-vcx/issues/933